### PR TITLE
AddingPowerSupport_cmd-shim.travis.yml

### PR DESCRIPTION
**Pull request by santosh653**
creating cmd-shim.travis.yml filewith proposed changes.,

### CODE REVIEW

File: travis-ymls/cmd-shim.travis.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : cmd-shim
+# Source Repo         : https://github.com/ForbesLindesay/cmd-shim.git
+# Travis Job Link     : https://travis-ci.com/github/santosh653/cmd-shim
```
Comment (seth-priya) [inline travis-ymls/cmd-shim.travis.yml:5]: looks like we have failing builds here?

Comment (santosh653) [reply travis-ymls/cmd-shim.travis.yml:5]: @seth-priya , there are failures on AMD64 windows distro, but the code is running fine on power.

Comment (seth-priya) [reply travis-ymls/cmd-shim.travis.yml:5]: can we exclude the failing builds in that case?

Comment (santosh653) [reply travis-ymls/cmd-shim.travis.yml:5]: @seth-priya Done the needful, Can you please confirm?

Comment (santosh653) [reply travis-ymls/cmd-shim.travis.yml:5]: @seth-priya Can you do the needful?

Comment (santosh653) [reply travis-ymls/cmd-shim.travis.yml:5]: @Siddhesh-Ghadi Can you please Review?
Thanks !!

Comment (santosh653) [reply travis-ymls/cmd-shim.travis.yml:5]: @gururajrkatti @Siddhesh-Ghadi Request you to please review?

**Review by seth-priya — APPROVED** [review]

**Pull request merged**